.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+# Pull Request Details
+
+Provide a general summary of your changes in the Title above
+
+## Description
+
+Describe your changes in detail
+
+## Related Issue(s)
+
+This project only accepts pull requests related to open issues in JIRA [1]
+If suggesting a new feature or change, please discuss it in an issue first
+If fixing a bug, there should be an issue describing it with steps to reproduce
+Please link to the issue here:
+
+## Motivation and Context
+
+Why is this change required? What problem does it solve?
+
+## How Has This Been Tested
+
+Please describe in detail how you tested your changes.  If the PR does not require testing
+please give a very good reason why that is the case.
+
+## Types of changes
+
+What types of changes does your code introduce? Put an `x` in all the boxes that apply:
+
+- [ ] Docs change / refactoring / dependency upgrade
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+## Checklist
```

Comment:
Could you please add a point to check whether all newly added dependencies are really required.

Reply:
@vvysotskyi Are we good to go here?